Updated streams-best-practices.md (#391)
osisoft/OCS-Docs: Content_Portal/add-organize-data/store-data/streams-best-practices.md

  <!-- I removed the following  and added a note in step 5 below telling them to leave it blank. --> <!-- 3. Use the stream name to provide a more readable name for the string than the stream ID itself. -->

- Use the stream description field, metadata, and tags to capture any other relevant information about the stream. <!--Not essential, since this bullet has sub bullets, but if possible, I suggest the following paragraph is in an indented paragraph, under the bullet -->This information is useful for searching for specific streams in the system, especially as systems become large. If possible, use consistent patterns in the description, metadata, and tags.

>    - <!--These 3 following are sub-bullets, right? -->Use metadata for information that follows a specific or consistent pattern, such as the location, manufacturer, and site. 

Reviewer: Just bringing it into your awareness that lines up to 20 are Laureen's content.

Author: Thanks Ji Won! I had fogotten that. Is it ok if I just add Laureen as a reviewer of this PR??

Reviewer: Of course! That sounds good, Joyce.

   - Use tags for simple information about the stream that does not adhere to any particular pattern.
   - Use the description field for longer descriptions of the stream and what it represents.

> ## Reading large volumes of data from streams by using API calls

Reviewer: ```suggestion
## Reading large volumes of data from streams with API

```
